common-build/rules.mk

```diff
@@ -217,10 +209,16 @@ dev autobuild: $(SPHINXAUTOBUILD)
 
 # Lint RST files and echo the results (for users and CI/CD logs)
 .PHONY: lint
+ifeq ($(UNAME),none)
+vale:
+	@ printf '\033[31mERROR: No rules to install Vale on your operating '
+	@ printf 'system or disabled by _no_vale file.\033[00m\n'
```

Comment:
> Tabs vs. spaces -- sigh ;].

Oh. I take full responsibility ;]. It looks like I didn't apply enough care to the `no_vale` case. Thanks for the fix!